main.tex

```diff
@@ -80,7 +81,7 @@ \section{Brückengleichrichter = \striche{Grätzbrücke}}\label{sec:bruckengleic
                 $U_{RM}$        & $\geq 1,5\cdot\sqrt{2}\cdot U_1$ (an den Dioden) \\
                 $U_{Br\,eff}$   & $=1,8\cdot 10^{-3}\cdot s \cdot\frac{I_{L-}}{C_L}$ \\
                 $U_{Br\,SS}$    & $=7\cdot 10^{-3}\cdot s \cdot\frac{I_{L-}}{C_L}$ \\
-                $C_L$           & $=\frac{I_L}{100}\frac{F}{A}$ (Standard: $\frac{\mu F}{mA}$)\\
+                $C_L$           & $=\frac{I_L}{100}\frac{F}{A}$ (Standard: $\frac{10\cdot\mu F}{mA}$)\\
```

Comment:
was genau sagt es mir, dass die Einheit meist 10µF/mA sind? mich verwirrt diese Angabe eher

Reply:
naja, so ist die umrechnung. so wie dein auto auch 8 L/100km verbraucht und nicht 8L/km (das wäre schlimm uiuiui), soll man den Kondensator mit 10uF/mA von I_L wählen, nicht mi t1uF/mA

Reply:
das ist nicht nur ne sache von einheiten (sonst könnten wir einfach 1F/A schreiben), sondern vom tatsächlichen zahlenwert

Reply:
okay. ich verstehe das Problem, finde es aber so nicht eindeutig. ich würde dann vlt eher die 10 vor den Bruch ziehen? oder sagen "Einheit: ..." ?

Reply:
NE EBEN NICHT EINHEIT

ob die 10 im oder vorm bruch steht ist mir relativ egal, hab die einfach nur willkürlich da hin gepackt
kannst auch 0.1 in den nenner schreiben wenn du lustig bist

Reply:
ich zieh die 10 davor, dann passt das